index.html

```diff
@@ -0,0 +1,186 @@
+              <button class="green-button">See Project</button>
+            </div>
+          </div>
+          <!-- Project Card Two -->
+          <div class="projects">
+            <div class="project-image"></div>
+            <div class="project-description">
+              <h4>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h4>
+              <ul class="languages">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button class="green-button">See Project</button>
+            </div>
+          </div>
+          <!-- Project Card Three -->
+          <div class="projects">
+            <div class="project-image"></div>
+            <div class="project-description">
+              <h4>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h4>
+              <ul class="languages">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button class="green-button">See Project</button>
+            </div>
+          </div>
+          <!-- Project Card Four -->
+          <div class="projects">
+            <div class="project-image"></div>
+            <div class="project-description">
+              <h4>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h4>
+              <ul class="languages">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+          <!-- Project Card five -->
+          <div class="projects">
+            <div class="project-image"></div>
+            <div class="project-description">
+              <h4>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h4>
+              <ul class="languages">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button class="green-button">See Project</button>
+            </div>
+          </div>
+          <!-- Project Card Six -->
+          <div class="projects">
+            <div class="project-image"></div>
+            <div class="project-description">
+              <h4>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h4>
+              <ul class="languages">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="about">
```

Comment:
- [ ] Please try to use `section` tag to wrap up your about page in order to follow HTML best practices which will make your codebase look professional